gremlin-console/src/main/java/org/apache/tinkerpop/gremlin/console/jsr223/DriverRemoteAcceptor.java

```diff
@@ -112,6 +113,8 @@ else if (shellEnvironment.getVariable(fileOrVar) != null) {
             return String.format("Configured %s", this.currentCluster) + getSessionStringSegment();
         } catch (final FileNotFoundException ignored) {
             throw new RemoteException("The 'connect' option must be accompanied by a valid configuration file");
+        } catch (NoHostAvailableException nhae) {
+            return "Host was not available - the Gremlin Console is making attempts to reconnect in accordance with the 'reconnectInterval' driver configuration but you may wish to close this remote and investigate the problem directly";
```

Comment:
Is this always the console?

Reply:
So there is no way to get the console test to pass if we throw exceptions at client initialization with no live host, so to get that test pass I referenced Stephen's old PR on 2569, and he had to make this change to the console, i.e. have it catch the exception and return the error message to the user, which I copied.